tests/posix_paths/test_posix_paths.py

```diff
@@ -0,0 +1,205 @@
+import json
+import os
+from datetime import datetime
+from pathlib import Path
+from typing import Optional
+
+import pytest
+
+import pystac
+from tests.conftest import get_data_file
+from tests.utils import ARBITRARY_BBOX, ARBITRARY_EXTENT, ARBITRARY_GEOM
+
+
+def check_link(link: Optional[pystac.Link]) -> None:
+    assert link is not None
+    href = link.get_target_str()
+    assert href is not None
+    assert "\\" not in href
+    print()
+    print(href)
+
+
+def test_create_item_from_absolute_posix_href() -> None:
+    # Should pass on both Windows and POSIX machines
```

Comment:
Not a useful comment, as we expect all tests to pass unless they have a platform-specific skip, right?